scRNA-seq-advanced/01-read_filter_normalize_scRNA.Rmd

```diff
@@ -185,17 +203,17 @@ Most values in this table are `NA`, because individual statistics were not calcu
 
 We can look at just the rows without `NA` values by selected the ones where the FDR (which we will use again soon), is not `NA`.
 
-```{r look at droplet stats}
+```{r droplet stats}
 # view rows where FDR is not `NA`
 droplet_df[!is.na(droplet_df$FDR), ]
 ```
 You will notice that some cells with high counts also have `NA` values for many statistics.
 In those cases, `NA` values are actually present _because_ of the high counts - `emptyDropsCellRanger()` automatically assumed cells were present, so they were also not tested. 
 
-
 Now we can filter our `raw_sce` object _by column_ to only keep the cells with a small FDR: those that are quite unlikely to be empty droplets. 
 
-```{r filter emptydrops}
+```{r filter emptydrops, live=TRUE}
+# filter droplets using `which` to prevent NA trouble
```

Comment:
I want to beef this up a tiny bit, although now I'm writing "`which` which", and we're back to https://github.com/AlexsLemonade/training-modules/pull/586#discussion_r1043887822! - 
But I do want to maybe get this idea across more explicitly?

```suggestion
# filter droplets using `which` which will ignore NAs
```